tests/Unit/FunctionCall/fcs_saxpy.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %hc -hc -amdgpu-function-calls %s -o %t.out && %t.out
+// RUN: %hc -hc -Wl,-hc-function-calls %s -o %t.out && %t.out
```

Comment:
`-hc-function-calls` should still work as a compiler switch?  In the use case with early finalize, the linker doesn't get invoked when we generate the isa.

Reply:
Yeah, that's where it'll end up, there's just a piece missing.